### DISCUSSION TITLE
How to disable automatic deployments for main branch?

### DISCUSSION THREAD

**Discussion by dace** [vercel/community#1680] [Help] [answered]
### Summary

Is it possible to disable automatic deployments whenever merges are made to `main` branch? I don't always want a merge to `main` to trigger a deploy. I'd prefer to manually trigger deployments for `main`. Is this possible?

**Comment by amyegan** [comment, marked as answer]
Hi @dace! You can use [`git.deploymentEnabled`](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#git.deploymentenabled) to disable automatic deployments from the `main` branch

**Reply by dace** [reply]
Awesome! Thank you @amyegan 🙏
